sip/sip-005-blocks-and-transactions.md

```diff
@@ -45,21 +45,24 @@ are paid for by an _account_.
 An _account_ is the logical entity that executes transactions.  A transaction's
 execution is governed by three accounts, which may or may not be distinct:
 
-* The **originating account** is the account that creates and sends the
-  transaction.  This is always an account owned by a user.  Each transaction is
-_authorized_ by its originating account.
-
-* The **paying account** is the account that is billed by the miner for the cost
-  of validating and executing the transaction.  This is usually an account owned
-by a user, but it may also be an account owned by a smart contract.  Its address
-is identified in each transaction separately from the originating account.
-
-* The **sending account** is the account that identifies _who_ is executing the
-  transaction.  Accounts may execute transactions on behalf of other accounts
-via the Clarity function `as-contract`.  Each transaction's initial sending
-account is its originating account -- i.e. the account that authorizes the
-transaction.  Smart contracts determine the sending account's principal using
-the `tx-sender` built-in function.
+* The **originating account** is the account that creates and sends
+  the transaction.  This is always an account owned by a user.  Each
+  transaction is _authorized_ by its originating account.
+
+* The **paying account** is the account that is billed by the miner
```

Comment:
This text is wrong (my fault).  The paying account is never owned by a smart contract; smart contracts can't pay for transactions.

Reply:
Okay -- I'll let you update that in your branch. I reverted the changes in these lines (I think it was just whitespace / wrapping.